### PR TITLE
replace feature.size in train_transforms and val_transforms

### PR DESCRIPTION
**Pull request by lilyaYAHIAOUI**
# What does this PR do?
this PR solve the issue #357 

Fixes # (issue)

## Who can review?

Feel free to tag members/contributors who may be interested in your PR.

### CODE REVIEW

File: examples/image_classification.ipynb [hunk 1]
```diff
@@ -948,7 +948,7 @@
     "normalize = Normalize(mean=feature_extractor.image_mean, std=feature_extractor.image_std)\n",
```
Comment (amyeroberts) [inline examples/image_classification.ipynb:948]: <p>Could you also add updates to the processing class: feature_extractor -&gt; image_processor</p><p> </p><pre class="ql-syntax" spellcheck="false">from transformers import AutoImageProcessor

image_processor = AutoImageProcessor.from_pretrained(model_checkpoint)
image_processor
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/notebooks/pull/359/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='36'/>

Comment (amyeroberts) [inline examples/image_classification.ipynb:948]: <pre class="ql-syntax" spellcheck="false">from transformers import AutoModelForImageClassification, AutoImageProcessor

repo_name = "nielsr/swin-tiny-patch4-window7-224-finetuned-eurosat"

image_processor = AutoImageProcessor.from_pretrained(repo_name)
model = AutoModelForImageClassification.from_pretrained(repo_name)
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/notebooks/pull/359/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='68'/>

Comment (amyeroberts) [inline examples/image_classification.ipynb:948]: <pre class="ql-syntax" spellcheck="false">encoding = image_processor(image.convert("RGB"), return_tensors="pt")
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/notebooks/pull/359/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='69'/>

**Review by amyeroberts — COMMENTED** [review]
Thanks a lot for updating this! 

I've added some comment with ReviewNB on additional places to update to make sure the deprecated feature extractors aren't used anymore

File: examples/image_classification.ipynb [hunk 2]
```diff
@@ -910,10 +910,10 @@
     }
```
Comment (amyeroberts) [inline examples/image_classification.ipynb:910]: <p>Update for the last two paragraphs:</p><p> </p><p>To make sure we (1) resize to the appropriate size (2) use the appropriate image mean and standard deviation for the model architecture we are going to use, we instantiate what is called an image processor with the&nbsp;<code>AutoImageProcessor.from_pretrained</code>&nbsp;method.</p><p>This image processor is a minimal preprocessor that can be used to prepare images for inference.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/notebooks/pull/359/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='35'/>

Comment (amyeroberts) [inline examples/image_classification.ipynb:910]: <p>Update to the paragraph:</p><p> </p><p>You might wonder why we pass along the&nbsp;<code>image_processor</code>&nbsp;as a tokenizer when we already preprocessed our data. This is only to make sure the image processor configuration file (stored as JSON) will also be uploaded to the repo on the hub.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/notebooks/pull/359/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='57'/>

Comment (amyeroberts) [inline examples/image_classification.ipynb:910]: <p>Update to code block:</p><p> </p><pre class="ql-syntax" spellcheck="false">from transformers import AutoModelForImageClassification, AutoImageProcessor

image_processor = AutoImageProcessor.from_pretrained("nielsr/my-awesome-model")
model = AutoModelForImageClassification.from_pretrained("nielsr/my-awesome-model")
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/notebooks/pull/359/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='64'/>

Comment (amyeroberts) [inline examples/image_classification.ipynb:910]: <p>Update to paragraph: </p><p> </p><p>We'll load the image processor and model from the hub (here, we use the&nbsp;<a href="https://huggingface.co/docs/transformers/model_doc/auto#transformers.AutoModelForImageClassification" target="_blank">Auto Classes</a>, which will make sure the appropriate classes will be loaded automatically based on the&nbsp;<code>config.json</code>&nbsp;and&nbsp;<code>preprocessor_config.json</code>&nbsp;files of the repo on the hub):</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/notebooks/pull/359/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='67'/>

Comment (amyeroberts) [inline examples/image_classification.ipynb:910]: <p>Update to paragraph: </p><p> </p><p>As we can see, it does not only show the class label with the highest probability, but does return the top 5 labels, with their corresponding scores. Note that the pipelines also work with local models and image processors:</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/notebooks/pull/359/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='76'/>

**Comment by amyeroberts** [comment]
@lilyaYAHIAOUI Thank for updating this! I just noticed some other places in the text descriptions to be updated too - apologies for not catching in the first pass. If you can add these, then I think we're good to merge :)

**Comment by lilyaYAHIAOUI** [comment]
@amyeroberts , I updated the text description

**Review by amyeroberts — APPROVED** [review]
@lilyaYAHIAOUI Thank you for iterating, adding this fix and propagating the changes! 

The part you might wish to change is the notebook outputs in examples/image_classification_albumentations.ipynb as the path currently points to your local environment.

**Comment by lilyaYAHIAOUI** [comment]
@amyeroberts I think this part is when we load ** AutoImageProcessor**. I modified it. Could you recheck it?

**Review by amyeroberts — APPROVED** [review]
Thanks for updating!

**Pull request merged**